src/unicode.hh

```diff
@@ -12,6 +12,7 @@ namespace Kakoune
 {
 
 using Codepoint = char32_t;
+const static Codepoint cp_zero = 0;
```

Comment:
using `0` or `Codepoint{0}` would be as good. I'd rather not add magic constants like that.

Reply:
I removed the constant. If there is any issue left - also formatting or style - let me know. I am not up to date with C++, it was 2005 when I last used it full-time.

Reply:
Clang on travis threw an error because of `0`, I switched to `Codepoint{0}`